test/e2e/deploy/deploy.go

```diff
@@ -420,6 +420,7 @@ func EnsureDeployment(deploymentName string) *Deployment {
 		cmd.Dir = os.Getenv("REPO_ROOT")
 		cmd.Env = os.Environ()
 		cmd.Env = append(cmd.Env,
+			"TEST_DEPLOYMENT_QUIET="+string("quiet"),
```

Comment:
This looks odd. Why the string cast, and why not simply "TEST_DEPLOYMENT_QUIET=quiet"?

Reply:
thanks, good point, I had over-complicated this by copying existing line. Changed it to
"TEST_DEPLOYMENT_QUIET=quiet" and pushed.